Day-11-Quiz-App/quizBuilder.js

```diff
@@ -122,8 +123,10 @@ class QuizBuilder {
     }
 
     _addInputElementsForOneQuestion(questionData) {
-        this._questionsContainer = document.getElementById("questionsContainer");
 
+        this._questionsContainer = document.getElementById("questionsContainer");
+        
+        const createHR = document.createElement("hr");
```

Comment:
variable naming could be improved here. The fact that it's created is onlys valid during until we call `document.createElement`. Afterwards it isn't.

I think it's better to call it by its function: `separatorHr`